app/src/components/fields/dynamicFieldsSection/dynamicField/dynamicField.jsx

```diff
@@ -40,30 +39,30 @@ export const DynamicField = ({
   );
   const fieldCommonProps = {
     name: field.id,
-    label: field.fieldName,
     required: field.required,
     disabled: field.disabled,
-    customBlock,
     ...rest,
   };
 
-  return FieldWrapper ? (
-    <FieldWrapper {...fieldCommonProps}>{fieldChildren}</FieldWrapper>
-  ) : (
-    <FormField
+  return (
+    <FieldElement
+      label={field.fieldName}
+      isRequired={field.required}
+      descriptionSecondary={field.description}
+      className={cx('dynamic-field')}
+      additionalInfo={customBlock}
+      normalize={(value) => value}
```

Comment:
Why do we need these props?

Maybe we will add an ability to provide them through the `DynamicField` props. What are your thoughts on that?

Reply:
They are the defaults from `FormField` component used previously.
And we already have such ability to override them defaults via `...fieldCommonProps`, f.e. [arrayField](https://github.com/reportportal/service-ui/blob/feature/EPMRPP-87336-Add-BTS-field-description/app/src/components/fields/dynamicFieldsSection/fields/arrayField.jsx#L66) use it.
I'm not sure if we need to mention all redux-form props here in prop types.